install/kubernetes/templates/_container_tetragon.tpl

```diff
@@ -11,7 +11,7 @@
     - /usr/bin/tetragon
 {{- end }}
   args:
-    - --config-dir=/etc/tetragon
+    - --config-dir=/etc/tetragon/tetragon.conf.d/
```

Comment:
Is this a convention? /etc/tetragon/conf.d seems better to me

Reply:
> Is this a convention? /etc/tetragon/conf.d seems better to me

Yes, but having explicit separate names could be consistent in future in case we add some specific tool with its conf that could load CRDs or policies into tetragon main agent.

Reply:
OK.